docs/cli-help.md

```diff
@@ -494,7 +494,7 @@ The set of commands that allows you to manage model and model versions.
   Role: `Vendor`
 
   Command: `dclcli tx model update-model --vid=<uint16> --pid=<uint16>  --from=<account>`
```

Comment:
I think we should we list all available fields in the command template, and not VID/PID only.